e2e/node/.env.example

```diff
@@ -1,24 +0,0 @@
-# Save a copy of this file as .env.test.local to prevent your credentials from
-# being committed.
-#
-# These credentials determine which Pod the end-to-end tests will write data to.
-# Generally, the tests should clean up after themselves, but it is advisable to
-# use a Pod created specifically for running these end-to-end tests.
-#
-# Note: The username present in the Pod resource URL should match the username
-# of the account used to generate the credentials - but just note that the
-# Inrupt Enterprise Solid Server (ESS) (or more precisely the ESS IdP Broker)
-# lower-cases the user's provided username when constructing their WebID, and
-# therefore all resources in that user's Pod, including their private root
-# resource, will contain that *lowercased* username in their URLs.
-#
-
-E2E_TEST_DEV_NEXT_IDP_URL=https://openid.dev-next.inrupt.com
-E2E_TEST_DEV_NEXT_VC_SERVICE=https://vc.dev-next.inrupt.com
-E2E_TEST_DEV_NEXT_REQUESTOR=https://id.dev-next.inrupt.com/devnextrequest
-E2E_TEST_DEV_NEXT_RESOURCE_OWNER=https://id.dev-next.inrupt.com/devnextrowner
-# Obtain these via https://openid.dev-next.inrupt.com/registration.html
-E2E_TEST_ESS_REQUESTOR_CLIENT_ID=5bc380bb-883f-468d-8275-68c903a75766
-E2E_TEST_ESS_REQUESTOR_CLIENT_SECRET=o9SCi3bDSmIqXyJqs6FE1smVMvZDgthK
-E2E_TEST_ESS_RESOURCE_OWNER_CLIENT_ID=57d036e2-5deb-4f71-b1c4-0b82327e9a29
-E2E_TEST_ESS_RESOURCE_OWNER_CLIENT_SECRET=e0fcdpcDsLLitXZTmJKFHhkuZa61B0sY
```

Comment:
0_0 these should never have been there. I'll revoke these credentials, generate new ones and update the secrets accordingly.

Reply:
oh dear